app/containers/MessageBox/RecordAudio.js

```diff
@@ -186,7 +186,7 @@ export default class RecordAudio extends React.PureComponent {
 					accessibilityLabel={I18n.t('Send_audio_message')}
```

Comment:
It was supposed to be done on GSoC, but it wasn't because other reasons.
This is not related to threads itself.
Just a code moved around.
Also, it's going to change soon when we rewrite this whole Audio stuff.